app/assets/javascripts/discourse/app/components/topic-footer-buttons.js

```diff
@@ -66,4 +69,19 @@ export default Component.extend({
   @discourseComputed("topic.message_archived")
   archiveLabel: (archived) =>
     archived ? "topic.move_to_inbox.title" : "topic.archive_message.title",
+
+  @discourseComputed(
+    "currentUser.auto_track_topics_after_msecs",
+    "topic.details.notification_level"
+  )
+  onboardingPopupContent(autoTrackDuration, notificationLevel) {
+    if (
+      autoTrackDuration > 0 &&
+      notificationLevel >= NotificationLevels.TRACKING
```

Comment:
The translation for the popup has the title "You are now tracking this topic", but we are showing it for watching as well, per `>= NotificationLevels.TRACKING`? That can be confusing. 

Can you also post a screenshot please, I'm not sure if we're showing a lot of copy here, because we do show a description here, example: 

<img width="865" alt="image" src="https://user-images.githubusercontent.com/368961/197860837-edd4c26a-25cf-44b4-8bec-e328c60bf747.png">